developer-docs-site/docs/guides/multisig-multiagent-transaction.md

```diff
@@ -0,0 +1,296 @@
+
+## Multiagent transaction
+
+A multiagent transaction requires multiple signers. A multiagent transaction must include the sender’s signature, along with a list of secondary signers’ signatures. For example, Alice minted some NFT tokens and she would like to transfer one of the NFTs to Bob. Token transfers require both the sender and receiver to sign the transaction. In this case, Alice prepares a multiagent transaction with her own signature as the sender signature, and she also needs Bob’s signature as the secondary signature for her transaction to be executed.
+
+The rest of this document describes the detailed steps of creating a multisig and a multiagent transaction.
+
+## Creating a multisig transaction
+
+1. Create a K-of-N multisig account.
+2. Create a raw transaction.
+3. Create a signing message.
+4. Create a multisig `Authenticator` and a `SignedTransaction`.
+5. Finally, serialize and submit the `SignedTransaction`.
+
+:::tip 
+
+The example code in this document is pseudo-code only, and is written in Typescript. The data types used are defined in [Creating a Signed Transaction](/guides/sign-a-transaction.md). 
+
+:::
+
+### Creating a K-of-N multisig account
+
+1. Generate a K-of-N multisig authentication key. Use `0x01` as the 1-byte multisig scheme identifier and your preferred threshold value for `K`. See [Signature scheme identifiers](/concepts/basics-accounts.md#signature-scheme-identifiers).
+2. Derive the account address from the authentication key.
+
+In the below example code, the `new Uint8Array([2, 1])` contains the threshold K, set to 2 and the 1-byte multisig scheme identifier, set to `1` .
+
+```typescript
+import * as SHA3 from 'js-sha3';
+const alicePubKey: Uint8Array = <KEY_BYTES>;
+const bobPubKey: Uint8Array = <KEY_BYTES>;
+const charliePubKey: Uint8Array = <KEY_BYTES>;
+// Concatenates the public keys, sets the threshold value K to 2 and the signature scheme identifier to 1
+const bytes = [...alicePubKey, ...bobPubKey, ...charliePubKey, ...new Uint8Array([2, 1])];
+const hash = SHA3.sha3_256.create();
+hash.update(Buffer.from(bytes));
+const authKey = new Uint8Array(hash.arrayBuffer());
+const multisigAccount = new AccountAddress(authKey);
+```
+
+### Creating a raw transaction
+
+```typescript
+function createRawTransaction(): RawTransaction {
+  const payload: ScriptFunction = {
+    module: {
+      address: hexToAccountAddress("0x01"),
+      name: "TestCoin"
+    },
+    function: "transfer",
+    ty_args: [],
+    args: [
+      BCS.serialize(hexToAccountAddress("0x02")), // recipient of the transfer
+      BCS.serialize_uint64(2), // amount to transfer
+    ]
+  }
+
+  // Assume the multisigAccount has test coins
+  return {
+    "sender": multisigAccount,
+    "sequence_number": 1n,
+    "max_gas_amount": 2000n,
+    "gas_unit_price": 1n,
+    "expiration_timestamp_secs": Math.floor(Date.now() / 1000) + 600,
+    "payload": payload,
+    "chain_id": 3
+  };
+}
+```
```

Comment:
similarly, if we can just point the the SDK, we could say that the multisigAccount can go anywhere the regular sender can?